src/Sulu/Bundle/AdminBundle/Resources/js/stores/MultiSelectionStore/MultiSelectionStore.js

```diff
@@ -10,19 +10,21 @@ export default class MultiSelectionStore<T = string | number, U: {id: T} = Objec
     resourceKey: string;
     locale: ?IObservableValue<string>;
     idFilterParameter: string;
+    requestParameters: {[string]: any};
```

Comment:
Wouldn't `{[string]: mixed}` be better for type safety? Or is that too much work to get it working?